api/jobs.yaml

```diff
@@ -458,6 +458,7 @@ definitions:
       - Failed
       - Succeeded
       - Aborted
+      - On Hold
```

Comment:
Can you include the regenerated `JobStatus` models for Cromwell and dsub?

Reply:
they weren't regenerated when I ran the swagger codgen, and it doesn't look like `dsub/jobs/models/job_status.py` or  `cromwell/jobs/models/job_status.py` specify possible values

Reply:
I would definitely not recommend putting a space in an enum value. I'd just go with `OnHold`.

Separately, what does OnHold mean and how does it relate to the other statuses? Depending on what it means, I'd suggest moving this up into its logical chronological order for readability (guessing before or after running?)

Reply:
The status in Cromwell has the space in it. If the `status` in JM is not exactly the same as the `status` submitted to Cromwell, where in the code is the best place to make that translation?

Reply:
I would add a helper and invoke here and here:

https://github.com/DataBiosphere/job-manager/blob/master/servers/cromwell/jobs/controllers/jobs_controller.py#L118
https://github.com/DataBiosphere/job-manager/blob/master/servers/cromwell/jobs/controllers/jobs_controller.py#L279

Reply:
what about updating and using the below existing code?

https://github.com/DataBiosphere/job-manager/blob/master/servers/cromwell/jobs/controllers/utils/task_statuses.py
https://github.com/DataBiosphere/job-manager/blob/master/servers/dsub/jobs/controllers/utils/job_statuses.py

Reply:
The first are task statuses. I still don't know what ON HOLD is, so I definitely don't know whether this is a task status, or just a workflow status (I couldn't find this status in the Cromwell docs either).

Good catch on the second. It's probably not going to matter since dsub doesn't have anything to map to ON_HOLD, but probably worth adding the reverse mapping ON_HOLD -> RUNNING just in case.

Reply:
@calbach "On Hold" is a pretty new workflow status, which indicates why the docs haven't been updated to include it: https://github.com/broadinstitute/cromwell/blob/develop/CHANGELOG.md#api

The first file is actually a mapping between JMUI status and Cromwell task execution statuses, @rsasch you might want to change it to `job_statuses.py` and add your own mapping to it.